src/mpid/ch4/src/ch4r_symheap.h

```diff
@@ -66,14 +66,15 @@ static inline int MPIDI_CH4R_check_maprange_ok(void *start, size_t size)
     for (i = 0; i < num_pages; i++) {
         rc = msync(ptr, page_sz, 0);
 
-        if (rc == -1) {
-            MPIR_Assert(errno == ENOMEM);
-            ptr += page_sz;
+        if(rc == -1) {
```

Comment:
The changes in this file mix whitespace with functional changes, and the whitespace changes do not follow the standard MPICH style. Please fixup the whitespace.